### ISSUE TITLE
Hide unimplemented features

### ISSUE THREAD

**Issue by WildWeazel** [C7-Game/Prototype#175]
Make all unusable buttons, menu options, etc invisible until they're functional. Maybe tie this to a build or runtime configuration- more discussion needed.

**Comment by QuintillusCFC** [comment]
Known items currently:

In-Game:

 - Rename unit
 - Civilopedia button
 - Most of the in-game menu options, only Quit works for now (and map takes you back to the map so you aren't stuck)

Main menu:

 - Tutorial
 - Hall of Fame
 - Preferences
 - Audio Preferences
 - A case could be made for Quick Start and New Game doing the same thing, but I'm inclined to leave them since one day Quick Start will do what both do now, and New Game will let you configure options for the game

I agree that just making them invisible would be ideal, since that should leave the layout in place.

**Comment by WildWeazel** [comment]
That was my thought too. If we tie their visibility to some global variable set in the release build we can hide them in releases only. My main concern is users reporting "broken" buttons.

**Comment by KulkoBSW** [comment]
I think I would like to work on this as my first issue, but I have a question.

Is there somewhere an example where I can see such an  global variable set? or would that be the first time we do something like this.

As for the Interface Buttons, I would set them to invisible right in the ready function? Or do you think that is a bad idea and it should be handled somewhere more central?

**Comment by esbudylin** [comment]
@KulkoBSW A good place for a global variable is the GlobalSingleton class, located in the C7/GlobalSingleton.cs file. This class represents the global state of the Godot application (see [Godot docs](https://docs.godotengine.org/en/stable/tutorials/scripting/singletons_autoload.html)). However, if we also want to make this variable build-configurable, we need to store is as a [project setting](https://docs.godotengine.org/en/stable/classes/class_projectsettings.html). 

For controlling button visibility, I think setting it at the button creation is a valid approach.

**Comment by WildWeazel** [comment]
I agree, project setting is ideal and checking at element creation should be fine. There's no need to change it at runtime, it's more of a debug vs release thing.
